jobs/registry/templates/config/registry.conf

```diff
@@ -29,8 +29,11 @@ http:
   relativeurls: <%= p('docker.registry.relativeurls') == true ? 'yes' : 'no' %>
 
 storage:
-  cache:
+  <% !if_p('docker.cache.disabled', 'true') do %>
```

Comment:
I just realized I broke backwards compatibility with this change.  When docker.cache.disabled is not specified this will not fill in the cache section at all.  I'll submit a followup to this with the appropriate changes.